views.py

```
@@ -0,0 +1,483 @@
+                        #                                                                                        flat=True).get()
+                        #     except:
+                        #         get_mail = User.objects.filter(username=request.POST.get('username')).values_list(
+                        #             'email', flat=True).get()
+                        #     return OneTimePasswordChange().post(request, get_mail)
+
+                        
+                        # Allow Reset password for all type of users when admin set the default password
+                        elif matchcheck and check_user_role in {'observer', 'admin', 'moderator', 'aspirant'}:
+                            try:
+                                get_mail = User.objects.filter(email=request.POST.get('username')).values_list('email', flat=True).get()
+                            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)